### DISCUSSION TITLE
FancyApps v5: Fancybox Caption moves to Footer

### DISCUSSION THREAD

**Discussion by GeronimoCH** [fancyapps/ui#593] [Q&A] [answered]
I have a gallery of images where I use FancyApps v5 Fancybox to show one of the pictures as single picture. Some of these pictures have a caption:
`<a href="http://xxx.yyy.local/content/6_Anderes/Bilder/Bild_02.jpg" data-fancybox="Anderes" data-caption="Guertel (Schnalle aus Neusilber)">`

If I click on a picture with a caption, the picture is shown with the caption just below the picture. If I make the window that shows the clicked picture less and less wide, suddenly the caption is no more shown just below the picture as expected but at the bottom of the window in the 'fancybox__footer'. In my FancyApps implementation I just uses fancybox.js and .css.

Question: Why does this happen and how can I avoid it? The main problem is, that the pictures captions are shown in the footer on a smartphone screen.

Thanks a lot for you help.
Hannes

**Comment by fancyapps** [comment, marked as answer]
Hi,

Fancybox5 tries to mimic the behavior of mobile apps by default. You can disable this feature using the `compact` option:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  compact: false
});  
```

https://jsfiddle.net/ewrmuo7d/

**Comment by GeronimoCH** [comment]
Hi Janis

Thanks a lot for your answer. You took less than one minute to answer my question! I find this just incredible.

Have a nice day
Hannes
